src/components/ResourceItem/tests/ResourceItem.test.tsx

```diff
@@ -87,6 +89,67 @@ describe('<ResourceItem />', () => {
     });
   });
 
+  describe('name', () => {
+    it('is used on the Checkbox label when item is not selected', () => {
+      const item = mountWithAppProvider(
+        <ResourceListContext.Provider value={mockSelectableContext}>
+          <ResourceItem
+            accessibilityLabel={accessibilityLabel}
+            id={itemId}
+            url="https://shopify.com"
+            name={name}
+          />
+        </ResourceListContext.Provider>,
+      );
+
+      const expectedLabel = `Select: ${name}`;
```

Comment:
Use the translation key instead so if it get's updated it automatically stays in sync.

```suggestion
      const expectedLabel = item
        .props()
        .i18n.translate('Polaris.ResourceList.Item.selectItem', {
        accessibilityLabel: name,
      });
```

Reply:
This doesn't seem to work. After some discussion with @BPScott, we opted to hardcode the string. Isolates the tests from the translate function.

Reply:
Ahh, gotcha. I thought we'd set up `mountWithAppProvider` like `web` set up `mountWithAppContext`.